Sources/_InternalTestSupport/XCTAssertHelpers.swift

```diff
@@ -43,8 +43,12 @@ public func XCTAssertEqual<T:Equatable, U:Equatable> (_ lhs:(T,U), _ rhs:(T,U),
     TSCTestSupport.XCTAssertEqual(lhs, rhs, file: file, line: line)
 }
 
+public func isRunninginCI(file: StaticString = #filePath, line: UInt = #line) -> Bool {
+    return (ProcessInfo.processInfo.environment["SWIFTCI_USE_LOCAL_DEPS"] != nil || ProcessInfo.processInfo.environment["CI"] != nil)
+}
+
 public func XCTSkipIfCI(file: StaticString = #filePath, line: UInt = #line) throws {
```

Comment:
The idiomatic way to do this with SwiftTesting would be a trait, something like this:

```swift
extension Trait where Self == Testing.ConditionTrait {
    static var skipInCI: Self {
        disabled("the test is being run on CI", ProcessInfo.processInfo.environment["SWIFTCI_USE_LOCAL_DEPS"] != nil || ProcessInfo.processInfo.environment["CI"] != nil)
    }
}
```